### PR TITLE
Feature :: Export All translations in Bulk

### PR DESCRIPTION
**Pull request by tsevillano**
* I add a default argument.
 * I extract to protected method How to generate translationsfile.
 * Export All data in bulk (And DB update!

### CODE REVIEW

File: src/Barryvdh/TranslationManager/Manager.php [hunk 1]
```diff
@@ -184,4 +185,12 @@ public function getConfig($key = null)
         }
     }
 
+    protected function exportFiles(&$locale, &$name, &$group)
```
Comment (barryvdh) [inline src/Barryvdh/TranslationManager/Manager.php]: Why all the references?

Comment (tsevillano) [inline src/Barryvdh/TranslationManager/Manager.php]: For not duplicate memory space 🤔  But I can change that!

**Comment by tsevillano** [comment]
ping @barryvdh

File: src/Barryvdh/TranslationManager/Manager.php (hunk 1 above)
Comment (barryvdh) [inline src/Barryvdh/TranslationManager/Manager.php]: can you change that?

**Comment by tsevillano** [comment]
Done! @barryvdh

**Comment by barryvdh** [comment]
But was the previous export not working? Or why change the logic?

**Comment by tsevillano** [comment]
Is to make it more optimal processing

Tomas Sevillano

On Mon, Nov 21, 2016 at 12:38 "Barry vd. Heuvel"

<
mailto:
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

But was the previous export not working? Or why change the logic?

—

You are receiving this because you authored the thread.

Reply to this email directly,
https://github.com/barryvdh/laravel-translation-manager/pull/151#issuecomment-261972468
, or

**Pull request closed without merging**